--- hdoc-1.4.0-orig/meson.build	2022-10-25 08:56:26.000000000 +0500
+++ hdoc-1.4.0/meson.build	2022-12-03 18:36:18.222184915 +0500
@@ -1,18 +1,11 @@
 project('hdoc', 'c', 'cpp', version: '1.4.0', default_options: ['cpp_std=c++20', 'warning_level=3'])
 
 dep_llvm = dependency('LLVM', include_type: 'system')
-clang_modules = [
-  'clangTooling',
-  'clangToolingInclusions',
-  'clangToolingCore',
-  'clangFrontend',
-  'clangAST',
-  'clangIndex',
-  'clangBasic',
-]
-dep_clang = dependency('Clang', include_type: 'system', method: 'cmake', modules: clang_modules)
 dep_zlib = dependency('zlib')
 
+cxx = meson.get_compiler('cpp')
+dep_clang = cxx.find_library('clang-cpp')
+
 # Disable RTTI
 add_project_arguments('-fno-rtti', language: 'cpp')
 
